tests/integration/test_issue_2235.py

```diff
@@ -0,0 +1,67 @@
+import os
+import subprocess
+import sys
+
+import pytest
+
+from pex.compatibility import PY3
+from testing import PY38, ensure_python_interpreter
+
+
+@pytest.mark.skipif(sys.version_info < (3, 8), reason="Flask 2.3.3 requires Python3.8+.")
+def test_standard_library_is_included(
+    tmpdir,  # type: Any
+    pex_project_dir,  # type: str
+):
+    # type: (...) -> None
+
+    # N.B.: The package script requires Python 3.
+    python = sys.executable if PY3 else ensure_python_interpreter(PY38)
+
+    package_script = os.path.join(pex_project_dir, "scripts", "package.py")
+    pex_pex = os.path.join(str(tmpdir), "pex")
+    subprocess.check_call(args=[python, package_script, "--pex-output-file", pex_pex])
+
+    subprocess.run(
+        [
+            sys.executable,
+            pex_pex,
+            "Flask==2.3.3",
+            "-o",
+            "flask.pex",
```

Comment:
You just leaked flask.pex to CWD (the root of the repo). Use a path formed from `tmpdir` instead.